Automattic/sensei
# Add exporter REST API and job

Part of #3049

### Changes proposed in this Pull Request

* Add (mostly empty) Export Job and Task classes. 
* Add REST API endpoint for exporter. 
* Add client integrating the API

### Testing instructions

* Starting the Exporter on the frontend should create a job, start it with the given content type, and then poll until it's done (with empty result).
* A Cancel button deletes the job
* Unit tests for the Export REST API do not contain the cases also covered by the importer tests, just for the endpoint that's different from the base class
